packages/shared/plugins/CursorHandler/core/utils/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const ZERO_WIDTH_SPACE = "\u200B";
+export const CARET = "\u2038";
+
+export const CURSOR_PLACEHOLDER_CHAR = ZERO_WIDTH_SPACE;
+
+export enum CursorMovementDirection {
+  LEFT = "left",
+  RIGHT = "right",
+}
+
+export enum charSelectionOffset {
```

Comment:
Nit if you are keeping consistent with the above enum, this should be PascalCase rather than camelCase (or change the first one to camelCase if you prefer).

Reply:
Thanks! fixed.